packages/server/tests/helpers/utils.ts

```diff
@@ -217,4 +241,77 @@ export class Utils {
 
     return await rpcServer.call(RelayCall.ETH_ENDPOINTS.ETH_GET_TRANSACTION_RECEIPT, [transactionHash], requestId);
   };
+  /**
+   * Creates an alias account on the mirror node with the provided details.
+   *
+   * @param {MirrorClient} mirrorNode The mirror node client.
+   * @param {AliasAccount} creator The creator account for the alias.
+   * @param {string} requestId The unique identifier for the request.
+   * @param {string} balanceInWeiBars The initial balance for the alias account in wei bars. Defaults to 10 HBAR (10,000,000,000,000,000,000 wei).
+   * @returns {Promise<AliasAccount>} A promise resolving to the created alias account.
+   */
+  static readonly createAliasAccount = async (
+    mirrorNode,
+    creator: AliasAccount,
+    requestId: string,
+    balanceInTinyBar: string = '1000000000', //10 HBAR
+  ): Promise<AliasAccount> => {
+    const signer = creator.wallet;
+    const accountBalance = Utils.tinyBarsToWeibars(balanceInTinyBar);
+    const privateKey = PrivateKey.generateECDSA();
+    const wallet = new ethers.Wallet(privateKey.toStringRaw(), signer.provider);
+    const address = wallet.address;
+
+    // send initial amount
+    await signer.sendTransaction({
```

Comment:
it's better to await these receipts because can cause false negative tests sometimes :confused: (account creation will be slower but the trade-off is acceptable I guess), but if we're prioritizing the speed, you can skip this comment :pray: